{{cookiecutter.repo_slug}}/.github/workflows/lock.yml

```diff
@@ -16,6 +16,18 @@ jobs:
           github-token: ${{ github.token }}
           issue-lock-inactive-days: "30"
           issue-lock-reason: ""
+          issue-comment: >
+            Issue closed and locked due to lack of activity.
+
+            If you encounter this same issue, please open a new issue and refer
+            to this closed one.
           pr-lock-inactive-days: "1"
           pr-lock-reason: ""
+          pr-comment: >
+            Pull Request closed and locked due to lack of activity.
+
+            If you'd like to build on this closed PR, you can clone it using
+            this method: https://stackoverflow.com/a/14969986
+
+            Then open a new PR, referencing this closed PR in your message.
```

Comment:
I feel strongly that when an issue or PR is locked, people who _subsequently_ stumble onto it (maybe because they're having the same issue) should be given some guidance as to what to do. This is my attempt to do this. I tried to find an official GitHub doc on cloning from a closed PR, but all I could find was the SO answer.